pandas/tests/frame/test_constructors.py

```diff
@@ -1424,6 +1424,20 @@ def test_from_dict_columns_parameter(self):
                 dict([("A", [1, 2]), ("B", [4, 5])]), columns=["one", "two"]
             )
 
+    @pytest.mark.parametrize(
+        "keys, values",
+        [([("a",), ("a",)], [1, 2]), ([("a",), ("b",)], [1, 2]), ([("a", "b")], [1])],
+    )
+    def test_constructor_from_dict_tuples(self, keys, values):
+        data_dict = {key: value for key, value in zip(keys, values)}
```

Comment:
Can you just parametrize dictionaries directly instead of doing this?

Reply:
Thanks @WillAyd - incorporated the change.